packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -308,8 +316,12 @@ export class AzureCommunicationCallWithChatAdapter implements CallWithChatAdapte
     return await this.chatAdapter.deleteMessage(messageId);
   }
   /* @conditional-compile-remove(file-sharing) */
-  public registerFileUploads(fileUploads: ObservableFileUpload[]): void {
-    this.chatAdapter.registerFileUploads(fileUploads);
+  public registerActiveFileUploads(files: File[]): FileUploadManager[] {
```

Comment:
why did we change the type of uploads from `ObservableFileUpload` to `File`

Reply:
Observable File Upload was an un-necessary interface that exposed attributes in the public API that were of no use to end users.
By using an HTML interface `File` we keep the API simple and eliminate the need for devs to deal with custom interfaces